src/game.cpp

```diff
@@ -99,6 +99,10 @@ void Game::init() {
 		//Threads.add([=](){ResMan.indexArchive(binFile, rmdpFile);});
 		ResMan.indexArchive(binFile, rmdpFile);
 	}
+
+	if (identifiers.empty())
+		throw std::runtime_error("No .rmdp files found in the set data path, exiting …");
```

Comment:
Now that i see it, could you replace … through three .?

Reply:
If you prefer that, sure